plugins/presentation/src/utils.ts

```diff
@@ -43,6 +43,8 @@ export function getUserId (): string {
   return platform.getMetadata(core.metadata.WhoAmI) as StringProperty
 }
 
+export type Query<T extends Doc> = Promise<QueryUpdater<T>>
```

Comment:
I"m not very good this actually, since with Promise it is clear what you could await on it, but not incase of Query.
It was planed to resolve this promise on first data was arrived.